src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -25,14 +25,18 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.UUID;
 
 /** Provides access to the data stored in Datastore. */
 public class Datastore {
 
-  private DatastoreService datastore;
-
+  private DatastoreService datastore; 
+  private static int longestMessage = 0;
+  private static HashMap<String, Integer> users = new HashMap<String,Integer>();
+  //hashmap how many times people have used it
```

Comment:
Are these comments related to the users hashmap above?
Typically we comment on something before we use/create it.
Also when writing a multi-line comment, use the 
/**
 * comment here
 * more comment
 */ 

Syntax whenever possible, bit easier to read. (Also most docs recommend full sentences when possible)

Besides that, not sure what arraylist you're referring to here.

Reply:
Sorry those comments were for me...forgot to delete them